### DISCUSSION TITLE
Multispeaker fine-tuning conditions

### DISCUSSION THREAD

**Discussion by 152334H** [152334H/DL-Art-School#19] [Ideas]
[So far](https://github.com/152334H/DL-Art-School/issues/1), I have failed to adequately train a fine-tuned model on a large variety of speakers. I notice that, even when I [undersample common speakers](https://github.com/152334H/DL-Art-School/issues/1#issuecomment-1435918388) to make the dataset more roughly diverse, I still notice a general failure of the resultant model to handle most speakers well -- even on the speakers with the maximum of 1k lines included!

Possible avenues for investigation:
- The issue of [cheater latents](https://github.com/152334H/DL-Art-School/issues/11)
- Fine-tuning other parts of the model? But then why would it work well enough for a single speaker, but not multiple speakers?
- Maybe a higher learning rate is needed? This sounds dumb, because the model was already hovering around a specific loss after 1k steps on 1e-5, but maybe it's stuck in some local minima?
- To check: is the dataset _actually_ randomised in order during training? I noticed that certain voices were better than others, maybe I just accidentally grouped all of them?
- Batch sizes? Maybe it'll train better if I just crank up grad accum + bs.
- Just train longer?

**Comment by Brugio96** [comment]
Hi @152334H  and thanks for your work, have you found a solution to this problem?

**Reply by 152334H** [reply]
I am quitting text-to-speech; blogpost coming Soon:tm:
